plugins/goosebit_forwarded_header/README.md

```diff
@@ -2,23 +2,27 @@
 
 Enables the use of standard Forwarded HTTP headers (RFC 7239). This has
 been introduced because uvicorn (in conjunction with gunicorn) does not
-support the feature (yet): https://github.com/encode/uvicorn/issues/2237
+support the feature (yet): [uvicorn#2237]
+
+[uvicorn#2237]: https://github.com/encode/uvicorn/issues/2237
 
 ## Setup
 
 ### Installation
 
 1. Load the plugin into poetry env (in goosebit):
-    ```bash
+
+    ```txt
     poetry install --with goosebit_forwarded_header
```

Comment:
As we now will have the plugins available on PyPI, I would like to replace this with a `pip` call. Is this even necessary with Poetry? When I run a plain `poetry install`, the plugins are installed as well:
```
  - Installing goosebit-forwarded-header (0.1.0 /home/ezra/src/goosebit/plugins/goosebit_forwarded_header)
  - Installing goosebit-simple-stats (0.1.0 /home/ezra/src/goosebit/plugins/goosebit_simple_stats)

```